prow/cmd/tot/main_test.go

```diff
@@ -193,11 +193,11 @@ func TestGetURL(t *testing.T) {
 		{
 			name: "fallback bucket - postsubmit",
 
-			jobName: "ci-test-infra-bazel",
+			jobName: "post-test-infra-test-all",
```

Comment:
not sure why this test depends on prow job config from this repo, but 🤷

Reply:
probably worth a TODO, tests for config/ are supposed to be under config/

Reply:
afaict these are not tests against job configs, they just need random job configs for unit testing this function. I'll decouple them after this PR

Reply:
#25553